scripts/checkfornewrun.bash

```diff
@@ -1,43 +1,94 @@
 #!/bin/bash
 
+set -u
```

Comment:
Why are there so many changes to this bash script in this PR?

Reply:
Mainly my linter calling out possible syntax errors caused by old bash code.

Reply:
But there seems to be a  lot of new code and not only linting stuff?

Reply:
there's novaseq stuff in here that does not belong, and the NIPT test is missing. I think something went wrong here. This is a 2500 script!

Reply:
Yep, I don't know what might have transgressed here. I updated with master and made sure I only added the call of the module.